openspec/changes/add-ansible-deployment/design.md

```
@@ -0,0 +1,285 @@
+│    │  └─ nix-portable-aarch64          │ ├─ flake.nix     │
+│    ├─ playbook.yml              ───┐   │ └─ ...           │
+│    ├─ roles/                       │   │                  │
+│    │  ├─ nix-portable/ ────────────┤   │ ~/              │
+│    │  └─ shelffiles/   ────────────┘   │ └─ .bashrc (opt) │
+│    └─ README.md                        │                  │
+└─────────────────┘                      └──────────────────┘
+```
+
+### Playbook Flow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)